lib/liquid/standardfilters.rb

```diff
@@ -370,7 +370,10 @@ def join(glue)
       end
 
       def concat(args)
-        to_a.concat args
+        unless args.respond_to?(:to_ary)
```

Comment:
Why here instead of in the "public" interface? Seems like a step too deep in the stack?